realm/src/androidTest/java/io/realm/RealmQueryTest.java

```diff
@@ -1072,35 +1072,36 @@ public void testSumValueForAllNullColumns() {
     public void testIsNullOnLinkField() {
         TestHelper.populateTestRealmForNullTests(testRealm);
 
+        // For the link with null value, query isNull on it's fields should return true.
         // 1 String
-        assertEquals(1, testRealm.where(NullTypes.class).isNull(
+        assertEquals(2, testRealm.where(NullTypes.class).isNull(
```

Comment:
Do we have to update the documentation? It seems that `isNull()` is behaving differently now (breaking change?).